src/chrome/content/rules/US-government.xml

```diff
@@ -167,79 +185,87 @@ Fetch error: http://wtr.nhtsa.gov/ => https://wtr.nhtsa.gov/: (7, 'Failed to con
 
 
 -->
-<ruleset name="US government (partial)" platform="mixedcontent" default_off='failed ruleset test'>
+<ruleset name="US government (partial)">
 
+	<target host="research.archives.gov" />
 	<target host="mtrics.cdc.gov" />
 	<target host="wwwn.cdc.gov" />
-	<target host="census.gov"/>
-	<target host="www.census.gov"/>
-	<target host="cms.gov"/>
-	<target host="*.cms.gov"/>
-	<target host="export.gov"/>
-	<target host="*.export.gov"/>
-	<target host="genome.gov"/>
-	<target host="www.genome.gov"/>
-	<target host="cms.hhs.gov"/>
-	<target host="www.cms.hhs.gov"/>
-	<target host="intelligence.house.gov"/>
+	<target host="census.gov" />
+	<target host="www.census.gov" />
+	<target host="cms.gov" />
+	<target host="assets.cms.gov" />
+	<target host="data.cms.gov" />
+	<target host="dnav.cms.gov" />
+	<target host="ehrincentives.cms.gov" />
+	<target host="eidm.cms.gov" />
+	<target host="marketplace.cms.gov" />
+	<target host="openpaymentsdata.cms.gov" />
+	<target host="portal.cms.gov" />
+	<target host="qhpcahps.cms.gov" />
+	<target host="questions.cms.gov" />
+	<target host="www.cms.gov" />
+	<target host="www1d.cms.gov" />
+	<target host="safeharbor.export.gov" />
+	<target host="genome.gov" />
+	<target host="www.genome.gov" />
+	<target host="cms.hhs.gov" />
+	<target host="applications.cms.hhs.gov" />
+	<target host="cob.cms.hhs.gov" />
+	<target host="www.cob.cms.hhs.gov" />
+	<target host="data.cms.hhs.gov" />
+	<target host="nppes.cms.hhs.gov" />
+	<target host="pecos.cms.hhs.gov" />
+	<target host="webinar.cms.hhs.gov" />
+	<target host="www.cms.hhs.gov" />
+	<target host="intelligence.house.gov" />
 	<target host="wwws.loc.gov" />
-	<target host="medicare.gov"/>
-	<target host="*.medicare.gov"/>
-	<target host="wtr.nhtsa.gov" />
-	<target host="usajobs.gov"/>
-	<target host="*.usajobs.gov"/>
-	<target host="postalinspectors.uspis.gov"/>
-	<target host="uspsoig.gov"/>
-	<target host="www.uspsoig.gov"/>
-
-
-	<securecookie host="^questions\.cms\.gov$" name=".*"/>
-	<securecookie host="^safeharbor\.export\.gov$" name=".*"/>
-	<securecookie host="^(?:.*\.)?(?:census|medicare|usajobs)\.gov$" name=".*"/>
-	<securecookie host="^intelligence\.house\.gov$" name=".*"/>
-	<securecookie host="^wtr\.nhtsa\.gov$" name=".+" />
+	<target host="medicare.gov" />
+	<target host="plancompare.medicare.gov" />
+	<target host="www.medicare.gov" />
+	<target host="mymedicare.gov" />
+	<target host="www.mymedicare.gov" />
+	<target host="usajobs.gov" />
+	<target host="help.usajobs.gov" />
+	<target host="login.usajobs.gov" />
+	<target host="my.usajobs.gov" />
+	<target host="www.usajobs.gov" />
+	<target host="postalinspectors.uspis.gov" />
+	<target host="uspsoig.gov" />
+	<target host="www.uspsoig.gov" />
```

Comment:
If you want HTTPS Everywhere to still use HSTS preloaded domains, you might go check out the [HSTS preload list](https://chromium.googlesource.com/chromium/src/+/master/net/http/transport_security_state_static.json) and take any `.gov` domains that have an `includeSubDomains` policy and add `*.` records for them.

Though, since HTTPS Everywhere is only available on Chrome and Firefox, and they both enforce HSTS preloading, you can **remove** rules from HTTPS Everywhere already covered by the preload list?